targets/targets.json

```diff
@@ -2430,7 +2430,7 @@
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-A9",
         "program_cycle_s": 2,
-        "extra_labels": ["RENESAS", "MBRZA1H"],
+        "extra_labels": ["RENESAS", "RZ_A1XX", "RZA1H", "MBRZA1H"],
```

Comment:
Would it be better to create RZ_A1XX target that the other would inherit from ? Thus device_has and all others would be also inherited. It could look like this:

```
- RZ_A1XX 
  - RZ_A1H
  - VK_RZ_A1H
```

Reply:
Not everything can be inherited. For example, in case of VK_RZ_A1H, "release_versions" is 2 only, and in case of GR_LYCHEE(will be next PR), there is not "Ethernet" function.
Thus, I will create "RZ_A1XX" target in the form of items that can be shared, and will inherit it in both RZ_A1H and VK_RZ_A1H.